compliance_checker/acdd.py

```diff
@@ -69,7 +69,7 @@ def check_recommended(self, ds):
     def check_suggested(self, ds):
         return [
             'contributor_name',
-            ('contributor_role', ['principalInvestigator', 'author']),
+            'contributor_role',
```

Comment:
Looks like it came from http://wiki.esipfed.org/index.php/Attribute_Convention_for_Data_Discovery_1-1
I feel like it was in 1-2, which I also think this checker was based on.  It's cool.